clients/src/test/java/org/apache/kafka/test/MockConsumerInterceptor.java

```diff
@@ -55,6 +58,11 @@ public void configure(Map<String, ?> configs) {
         Object clientIdValue = configs.get(ConsumerConfig.CLIENT_ID_CONFIG);
         if (clientIdValue == null)
             throw new ConfigException("Mock consumer interceptor expects configuration " + ProducerConfig.CLIENT_ID_CONFIG);
+
+        CONFIG_COUNT.incrementAndGet();
+        if (CONFIG_COUNT.get() == THROW_CONFIG_EXCEPTION_THRESHOLD.get()) {
+            throw new ConfigException("Kafka producer creation failed. Failure may not have cleaned up listener thread resource.");
```

Comment:
It seems like the failure message here is hinting that we try to create a Kafka producer in this interceptor, but there isn't much else in the class to go along with that.

Could we use a more generic message like "Failed to instantiate interceptor (reached throw-on-config threshold)"?

Reply:
Done!